pyproject.toml

```diff
@@ -23,8 +23,9 @@ dependencies = [
     "sentry-sdk >=2.1.0",
     "mixpanel >=4.10.1",
     "pydantic >=2.11.0",
+    "python-dateutil>= 2.9.0",
```

Comment:
This is already a required dependency of `pandas`, this makes it an explicit dependency.